stock_request/models/stock_request_order.py

```diff
@@ -317,6 +317,16 @@ def _check_location_company(self):
                 )
             )
 
+    @api.constrains("stock_request_ids")
+    def _check_location_empty_stock_request_ids(self):
+        if any(not request.stock_request_ids for request in self):
+            raise ValidationError(
+                _(
+                    "It is not possible to set empty stock request orders (maybe "
+                    "you should cancel it)."
```

Comment:
AFAIK, you can't cancel the order, so the message is misleading.

Reply:
I think it is possible. Create a stock request order with lines and save. Try to delete lines and save (error, if you want that you should cancel it).

Reply:
No, Víctor, you can only cancel orders with requests, not empty ones, according to the computed field.